formats/hocon/src/test/kotlin/kotlinx/serialization/hocon/HoconValuesTest.kt

```diff
@@ -103,4 +130,31 @@ class HoconValuesTest {
             assertEquals(listOf(null, WithNullable(10, "bar")), i3)
         }
     }
+
+    @Test
+    fun `deserialize list of integer string values`() {
+        val configString = """i1 = [ "1","3" ]"""
+        val obj = deserializeConfig(configString, WithList.serializer())
+        with(obj) {
```

Comment:
It seems that `with` is unnecessary here, sine we access only one property

Reply:
True. I removed it.